### PR TITLE
Fold in basic flow for login/reg and minimal checkout.

### PR DESCRIPTION
**Pull request by mcstover**

### CODE REVIEW

**Comment by mcstover** [comment]
@emuvente @eddieferrer Please review at your leisure. These are all isolated changes in prep for some demo recordings were doing today. The concept of InContextCheckout that covers a minimal basket and its required validations might be of interest.

File: src/pages/LandingPages/CorporateCampaign/InContextCheckout.vue [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+		};
+	},
+	computed: {
+		registerOrLoginHref() {
+			if (typeof window !== 'undefined') {
+				return `/ui-login?force=true&doneUrl=${window.location.href}`;
+			}
+			// TODO: map using route for ssr
+			return '/ui-login?force=true';
```
Comment (emuvente) [inline src/pages/LandingPages/CorporateCampaign/InContextCheckout.vue:93]: For both client and server you can use
```
return `/ui-login?force=true&doneUrl=${this.$route.path}`
```

Comment (mcstover) [reply src/pages/LandingPages/CorporateCampaign/InContextCheckout.vue:93]: Cool thanks. I need to make sure all query params will flow through as well and will work that out with the router data.

Comment (emuvente) [reply src/pages/LandingPages/CorporateCampaign/InContextCheckout.vue:93]: Good point, you might need to do `encodeURIComponent(this.$route.fullPath)`

**Review by emuvente — COMMENTED** [review]
👍

**Pull request merged**